style.css

```diff
@@ -0,0 +1,225 @@
+/*==========================================================
+    FONTS
+==========================================================*/
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700&display=swap');
+/*==========================================================
+    VARIABLES
+==========================================================*/
+:root {
+    --news-red: #f72210;
+    --ethic-white: #fff;
+}
+/*==========================================================
+    GENERAL TAGS
+==========================================================*/
+body {
+    font-family: Arial,Helvetica,sans-serif;
+}
+
+.hid {
+    display: none;
+}
+
+body.container {
+    max-width: 1290px;
+    overflow-x: hidden;
+}
+
+h1, h2, h3, h4, h5 {
+    font: bold 19px/19px "Roboto Condensed",sans-serif;
+}
+
+p {
+    margin: 0;
+}
+
+ul, ol {
+    margin: 0;
+    list-style: none;
+}
+
+.fs-1-3 {
+    font-size: 1.3rem;
+}
+
+.txt-red {
+    color: var(--news-red)
+}
+
+.max-1290 {
+    max-width: 1290px;
+}
+
+section.max-1290 {
+    max-width: 1600px !important;
```

Comment:
#### `REQUIRED`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. 

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

> Make sure you DON’T make [common mistakes](https://speckyboy.com/good-bad-css-practices/) in your CSS styles.

> **Recommendation:** You should only use the !important rule when overriding styles in a user style sheet, overriding 3rd party code and inline styles as well as utility classes.

Reply:
Done. The important tag ended up not being needed, removed.